garden_ai/pipelines.py

```diff
@@ -105,39 +105,28 @@ class PipelineMetadata(BaseModel):
     papers: List[Paper] = Field(default_factory=list)
 
 
-class RegisteredPipeline(BaseModel):
+class RegisteredPipeline(PipelineMetadata):
```

Comment:
Now this is a strict superset of PipelineMetadata, hence not needing to retype all the fields